.travis.yml

```diff
@@ -15,8 +15,8 @@ install:
   - rvm use 2.2.2 --default
   - rvm @global do gem install bundler
   # Kalibro Install script
-  - git clone https://github.com/mezuro/kalibro_install.git -b v3.3
-  - ANALIZO_VERSION= CACHE_DIR=$PWD/cache bash ./kalibro_install/install.sh
+  - git clone https://github.com/mezuro/kalibro_install.git -b v3.6
+  - ANALIZO_VERSION=none KALIBRO_PROCESSOR_VERSION=hotspot_metrics_list CACHE_DIR=$PWD/cache bash ./kalibro_install/install.sh
```

Comment:
We need to release another version of kalibro processor and update kalibro_install